tests/measureSources.py

```diff
@@ -237,15 +237,20 @@ def testPixelFlags(self):
         edge = mask.getPlaneBitMask('EDGE')
         bad = mask.getPlaneBitMask('BAD')
         nodata = mask.getPlaneBitMask('NO_DATA')
+        mask.addMaskPlane('CLIPPED')
+        clipped = mask.getPlaneBitMask('CLIPPED')
         mask.set(0)
         mask.set(20, 20, sat)
         mask.set(60, 60, interp)
         mask.set(40, 20, bad)
         mask.set(20, 80, nodata)
+        mask.set(30, 30, clipped)
         mask.Factory(mask, afwGeom.Box2I(afwGeom.Point2I(0,0), afwGeom.Extent2I(3, height))).set(edge)
         x0, y0 = 1234, 5678
         exp.setXY0(afwGeom.Point2I(x0, y0))
         control = measBase.PixelFlagsControl()
+        # Change the configuration of control to test for clipped mask plane DM-2915
```

Comment:
I queried this before, but: is it necessary or helpful to keep repeating the string "DM-2915" in the comments? If every line of code came with a note of the ticket which caused it to be added, it'd get unreadable pretty fast. (Plus, `git blame` would be out of a job.)